packages/aws-cdk-lib/aws-cloudfront/lib/distribution.ts

```diff
@@ -1082,6 +1082,14 @@ export interface AddBehaviorOptions {
    * @see https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/PrivateContent.html
    */
   readonly trustedKeyGroups?: IKeyGroup[];
+
+  /**
+   * Enables your CloudFront distribution to receive gRPC requests and to proxy them directly to your origins.
+   *
+   * @default - gRPC will be disabled
```

Comment:
The reason I don't write it as `@default false` is to mislead users into thinking that a value of `false` will be set into `enabled`. In fact, the object will be undefined in the first place.

The code:
```ts
      grpcConfig: this.props.enableGrpc !== undefined
        ? {
          enabled: this.props.enableGrpc,
        }
        : undefined,
```